robot_smach_states/examples/guide-node

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
```

Comment:
Is this file a good example of how the guide state machine should be used? Why is this here?